src/main/java/blusunrize/immersiveengineering/common/EventHandler.java

```diff
@@ -95,6 +94,8 @@
 	public static final Queue<Pair<DimensionType, BlockPos>> requestedBlockUpdates = new LinkedList<>();
 	public static final Set<TileEntity> REMOVE_FROM_TICKING = new HashSet<>();
 
+	private static final ResourceLocation tall_grass_drop = new ResourceLocation("minecraft", "blocks/tall_grass");
```

Comment:
Consider naming this constant in all-caps

Reply:
Done.
Did not touch requestedBlockUpdates analogously, though, to prevent muddying the PR.

Reply:
Naming what's called a constant (members with public, static and final (order of static and final can be interchanged)) in all uppercase spaced with underscore is suggested in the Java coding and naming conventions.

Reply:
...yeah, I mean, that's kinda why I requested the change, lol.
You're a bit late there.